charts/argocd-notifications/Chart.yaml

```
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Initialize Changelog"
+    - "[Added]: Support of 'defaultTriggers'"
```

The old changelog entry needs to be removed
```suggestion
    - "[Added]: Support of 'defaultTriggers'"
```